Streetcode/Streetcode.BLL/MediatR/Team/Position/Delete/DeleteTeamPositionHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentResults;
+using MediatR;
+using Streetcode.BLL.Interfaces.Logging;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+namespace Streetcode.BLL.MediatR.Team.Position.Delete;
+
+public class DeleteTeamPositionHandler : IRequestHandler<DeleteTeamPositionCommand, Result<int>>
+{
+    private readonly IRepositoryWrapper _repository;
+    private readonly ILoggerService _logger;
+
+    public DeleteTeamPositionHandler(IRepositoryWrapper repository, ILoggerService logger)
+    {
+        _repository = repository;
+        _logger = logger;
+    }
+
+    public async Task<Result<int>> Handle(DeleteTeamPositionCommand request, CancellationToken cancellationToken)
+    {
+        var positionToDelete =
```

Comment:
You can reduce and optimize this code by making just one database query. This option is valid, but it is better to use BulkDelete
like this 
_dbContext.Positions.Where(x => x.Id == id).ExecuteDelete();